web/controllers/pdf_conversion_controller.ex

```diff
@@ -1,26 +1,21 @@
 defmodule DailyRoutine.PdfConversionController do
```

Comment:
Name of the controller is confusing. It should probably be something like `PdfRequest` or something to do with sending an email for example.